dbms/src/Access/AccessRightsContext.cpp

```diff
@@ -157,9 +157,9 @@ void AccessRightsContext::setUser(const UserPtr & user_) const
         current_roles.reserve(params.current_roles.size());
         for (const auto & id : params.current_roles)
         {
-            if (user->granted_roles.contains(id))
+            if (user->granted_roles.find(id) != user->granted_roles.end())
```

Comment:
I don't appreciate this.

Reply:
Replaced with a `count()` for now. Don't think, that `#ifdef`s will be better.